packages/node/src/integrations/anr/worker.ts

```diff
@@ -173,7 +173,7 @@ if (options.captureStackTrace) {
         {
           // Grab the trace context from the current scope
           expression:
-            'const ctx = __SENTRY__.acs?.getCurrentScope().getPropagationContext() || {}; ctx.traceId + "-" + ctx.spanId + "-" + ctx.parentSpanId',
+            'const __sentry_ctx = __SENTRY__.acs?.getCurrentScope().getPropagationContext() || {}; __sentry_ctx.traceId + "-" + __sentry_ctx.spanId + "-" + __sentry_ctx.parentSpanId',
           // Don't re-trigger the debugger if this causes an error
```

Comment:
Whatever context this is evaluated in, the next time the same Runtime.evaluate is used (assuming support for multiple ANR captures is added), won't this potentially just lead to the same conflicting variable issue due to it now being defined from the last time Runtime.evaluate defined it? Here was my solution:

```typescript
const t = Date.now()
// Evaluate a script in the currently paused context
session.post('Runtime.evaluate', {
	// Grab the trace context from the current scope
	expression: `const ctx_${t} = __SENTRY__.acs?.getCurrentScope().getPropagationContext() || {}; ctx_${t}.traceId + "-" + ctx_${t}.spanId + "-" + ctx_${t}.parentSpanId`,
	// Don't re-trigger the debugger if this causes an error
	silent: true,
}, (err, param) => {
```

Just using `var` instead of `const` with your `__sentry_ctx ` would also work?

Reply:
Yep good point!